lessons/taquito/02/02.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Signing our keys
+chapter: Chapter 2
+slug: module-04/chapter-02
+filterBy: lesson-4
+editor:
+  language: javascript
+  startingCode: |
+    // import { importKey } from "@taquito/signer";
```

Comment:
Shouldn't we remove this comment as we have already mentioned that importKey function is already imported in the background.